src/app/schemas/generation_template.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+from pydantic import BaseModel
+from pydantic.schema import datetime
+
+
+class GenerationTemplateBase(BaseModel):
+    name: str
+    description: Optional[str]
+    content: str
+
+
+class GenerationTemplateCreate(GenerationTemplateBase):
+    pass
+
+
+class GenerationTemplateUpdate(GenerationTemplateBase):
+    name: Optional[str]
+    description: Optional[str]
+    content: Optional[str]
+
+
+# Properties shared by models stored in DB
+class GenerationTemplateInDBBase(GenerationTemplateBase):
+    id: int
```

Comment:
We should already move the id to the `GenerationTemplateBase` class like in the other schemas now. If the id is only present in the database representation class, we have no way to return the frontend the id of an object of this type. Therefor they could never know, which id they have to put for an update, or delete.

Reply:
Addressed